server/models/model.boardgames.js

```diff
@@ -51,4 +51,15 @@ exports.selectReviewCommentsById = (review_id) =>{
     .then((result =>{
         return result.rows;
     }));
-} 
\ No newline at end of file
+} 
+
+exports.insertCommentByReviewId = (review_id, newComment) =>{
+    let sql = `INSERT INTO comments (review_id, body, author)
```

Comment:
const > let unless we need it to be a let (we might later on but for now we don't)